lib/objects/objectsInMemFileDB.js

```diff
@@ -2569,6 +2574,52 @@ class ObjectsInMemoryFileDB extends InMemoryFileDB {
             this.writeTimer = null;
         }
     }
+
+    isEquivalentValue(a, b, ignoreField) {
+        // Create arrays of property names
+        if (a === null || a === undefined || b === null || b === undefined) {
+            return (a === b);
+        }
+        const aProps = Object.getOwnPropertyNames(a);
+        const bProps = Object.getOwnPropertyNames(b);
+
+        const lengthModA = a[ignoreField] !== undefined ? 1 : 0;
+        const lengthModB = b[ignoreField] !== undefined ? 1 : 0;
```

Comment:
Would it make sense to provide an array of ignored fields instead of a single one?

Reply:
Guess Bluefox answered that one for me :)